src/liteclient.cc

```diff
@@ -586,13 +586,19 @@ void LiteClient::reportAppsState() {
     return;
   }
   const auto apps_state{compose_pacman->getAppsState()};
-
   if (apps_state.isNull()) {
     LOG_WARNING << "Failed to obtain Apps state, skipping sending it to Device Gateway";
     return;
   }
+  if (!apps_state_.isNull() && apps_state_.isMember("apps") && apps_state.isMember("apps") &&
```

Comment:
looks good. merge it like this if you want, but i wonder if this would be a little cleaner if you moved these checks into `compareAppsStats`

Reply:
makes sense.